olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationSettings.h

```diff
@@ -77,6 +77,12 @@ struct AUTHENTICATION_API AuthenticationSettings {
    * in-memory cache.
    */
   size_t token_cache_limit{100u};
+
+  /**
+   * @brief Use local system time in authentication request rather than requesting
+   * time from authentication server.
+   */
+  bool use_local_time{false};
```

Comment:
I would not call it local time. Local time usually is not UTC and might be confusing, as we do not use actual local time but system time. I would name it `use_system_time`.